src/PrestaShopBundle/Controller/Admin/Configure/AdvancedParameters/EmployeeController.php

```diff
@@ -360,10 +361,16 @@ public function editAction($employeeId, Request $request)
             $result = $this->getEmployeeFormHandler()->handleFor((int) $employeeId, $employeeForm);
 
             if ($result->isSubmitted() && $result->isValid()) {
+                $employeeId = $result->getIdentifiableObjectId();
+
+                if ($contextEmployeeProvider->getId() === $employeeId) {
+                    $this->renewAuthenticationCredentials($employeeId);
```

Comment:
since the credentials were changed in the form, we need to update them in the cookie. 

It was done in legacy as well: https://github.com/PrestaShop/PrestaShop/blob/1.7.5.x/controllers/admin/AdminEmployeesController.php#L632